### ISSUE TITLE
Missing string "Positive charges do not move."

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#357]
Induced charge is in a lot of places and different in different places, too (sorry!) There will be a few issues about induced charge. For simplicity, I will keep them separate.

**Missing "Positive charges do not move."**
- [x] Please add, "Positive charges do not move." at end of induced charge description in Scene Summary, and
- [x] at the end of the Wall's Dynamic description. 

I missed this string in some of the examples in the Two-balloon Design Document and just added it. (sorry!)

**Comment by jessegreenberg** [comment]
@terracoda this was done in the above commits. Closing.
